components/cli/cmd/cellery/cellery.go

```diff
@@ -56,7 +59,7 @@ func newCliCommand() *cobra.Command {
 }
 
 func main() {
-	logFileDirectory := filepath.Join(util.UserHomeDir(), ".cellery", "logs")
+	logFileDirectory := filepath.Join(util.UserHomeDir(), constants.CELLERY_HOME, "logs")
```

Comment:
Shall we change the value of constants.CELLERY_HOME to ->
CELLERY_HOME := filepath.Join(util.UserHomeDir(), ".cellery")

Reply:
The constant is already defined in the code base. I think we should do the change at once for the whole code base at once later.